pymc_experimental/model_builder.py

```diff
@@ -187,22 +192,29 @@ def load(cls, fname):
 
         filepath = Path(str(fname))
         idata = az.from_netcdf(filepath)
-        self = cls(
-            json.loads(idata.attrs["model_config"]),
-            json.loads(idata.attrs["sampler_config"]),
-            idata.fit_data.to_dataframe(),
+        if "sampler_config" in idata.attrs:
```

Comment:
I think we should assume that it's always there.

Reply:
ah okey. Actually you're right, I don't know why I assumed that if we agreed to make he sampler config and model config as optional then we'd not assign empty attribute spot to sampler config. But I agree that to make it more logically continuous we should add whatever is there, whether is a valid config dict or just an empty dict.